crates/re_arrow_store/src/store_write.rs

```diff
@@ -168,27 +141,22 @@ impl DataStore {
             }
         }
 
-        // This is valuable information, even for a timeless timepoint!
+        // This is valuable information, even for a timeless !
         self.messages.insert(*msg_id, time_point.clone());
 
         Ok(())
     }
 
-    fn insert_timeless_row(
+    fn _insert_timeless_row(
```

Comment:
why the underscore prefix? is this unused but will become used soon? If so add a comment about it instead, preferably with a `#[allow(dead_code)] // this will be revived when …` instead of using the underscore prefix

Reply:
`_insert_row` & `_insert_timeless_row` are private helpers for `insert_row`; the underscore denotes that

Reply:
That's… surprising.

I would prefer a `_helper` suffix instead, as a `_` prefix means something else in Rust ("fine if unused", among other things). I can already see it is private by the lack of `pub`